framework/test/unit/modules/interoperability/mainchain/commands/recover_message.spec.ts

```diff
@@ -291,7 +293,9 @@ describe('MessageRecoveryCommand', () => {
 			];
 			ccmsEncoded = ccms.map(ccm => codec.encode(ccmSchema, ccm));
 			transactionParams.crossChainMessages = [...ccmsEncoded];
-			transactionParams.idxs = [2 ** terminatedChainOutboxSize + terminatedChainOutboxSize];
+
+			const indexLength = getHeight(terminatedChainOutboxSize);
+			transactionParams.idxs = [2 ** indexLength + terminatedChainOutboxSize];
```

Comment:
As it is not straight forward to see what's happening here, you could leave a comment like this one:

"Choose the index such that the position of the ccm in the outbox tree is at terminatedChainOutboxSize, i.e. just outside of the tree. Note that a 1 has to be prepended to the binary representation of the index, which is done by adding 2**indexLength. See https://github.com/LiskHQ/lips/blob/main/proposals/lip-0031.md#proof-serialization."

Up to you.

Reply:
Sounds good, will add the above comment to `appendPrecedingToIndices` in this PR 👍 
https://github.com/LiskHQ/lisk-sdk/pull/8847